src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -594,29 +594,48 @@ internal static IDisposable OnLoaded(this UIView uiView, Action action)
 			}
 
 			Dictionary<NSString, NSObject> observers = new Dictionary<NSString, NSObject>();
-			ActionDisposable? disposable = new ActionDisposable(() =>
+			ActionDisposable? disposable = null;
```

Comment:
The modifications here all were necessary to fix `FlyoutPage` so it would fire `Loaded`. The Unit Tests use `Loaded` to know when to proceed with the tests.